src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -701,6 +704,10 @@ public boolean getUseSessionTimezone() {
     return useSessionTimezone;
   }
 
+  public boolean getGetDateUseNullTimezone() {
+    return getDateUseNullTimezone;
```

Comment:
does this field has to be name this way? no way to avoid this `getGet`

Reply:
> JDBC_GET_DATE_USE_NULL_TIMEZONE is the name of the property that influences this field. I don't like this name, but I think it should be similar to the original property.

https://github.com/snowflakedb/snowflake-jdbc/pull/1877#discussion_r1734161194

Reply:
shame, but sure, let's leave it as is.